book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Christopher Patton"]
```

Comment:
Not that I don't appreciate your contributions but could we change this to "The authors" or "The contributors" or remove it altogether if it's optional?

Reply:
Oops, that was added auto-magically

Reply:
Done.